Gemfile.lock

```diff
@@ -2,10 +2,11 @@ GEM
   remote: https://rubygems.org/
```

Comment:
Please pardon me for asking this again.  I'm sure I asked this before about the `Gemfile.lock` file in this or another Ruby project, but I don't remember what was the response.  I looked through my GH activity and I didn't see my older comment.

So, why is `Gemfile.lock` being included in the repo?  It doesn't seem appropriate.  If some other developer comes along and clones this project for the first time and tries to install the dependencies, won't the lock file prevent it?  Similarly, I believe it would cause trouble if I included the `composer.lock` file in a PHP project.  The other developer would need to delete the lock file before the dependencies could be installed.